Gopkg.toml

```diff
@@ -37,10 +37,6 @@ required = [
   name = "github.com/golang/protobuf"
   version = "1.2.0"
 
-[[constraint]]
-  name = "github.com/spf13/cobra"
```

Comment:
need vendor folder update if vendoring is reintroduced?

Reply:
Yeah, but the lock should not be changed because we do not use the package. Thus the definition here will be ignored by dep now.